LDAR_Sim/src/programs/method.py

```diff
@@ -219,16 +219,10 @@ def deploy_crews(self, workplan: Workplan, weather, daylight) -> CrewDeploymentS
         for crew in self._crew_reports:
             # TODO : if method is daylight sensitive, check for max daylight
             crew.day_time_remaining = day_time_remaining
+            crew.deployed = False
             priority_queue.put((-crew.day_time_remaining, crew.crew_id, crew))
 
-        # If the cost type for the method is per day, calculate the deployment cost for day
-        # based off the number of crews being deployed
-        if self.cost_type == self.PER_DAY_COST:
-            if self._deployment_type == pdc.Deployment_Types.STATIONARY:
-                deploy_stats.deployment_cost = self.cost * len(workplan.site_survey_planners)
-            else:
-                deploy_stats.deployment_cost = self.cost * len(self._crew_reports)
-
+        workplans_to_add = []
```

Comment:
Nit: This needs a type annotation

Reply:
Style and clarity: Also I think renaming to something like unfinished surveys might be more clear